src/metabase/api/database.clj

```diff
@@ -31,7 +31,8 @@
             [schema.core :as s]
             [toucan
              [db :as db]
-             [hydrate :refer [hydrate]]])
+             [hydrate :refer [hydrate]]]
+            [clojure.java.io :as io])
```

Comment:
does io need to be imported here?

Reply:
also ns sorting :wink:

Reply:
No, the refactor-nrepl plugin was being...helpful there, I removed it (and sorted it for good measure)